GeoIP2.UnitTests/DatabaseReaderTests.cs

```diff
@@ -9,6 +9,7 @@
 using System.IO;
 using System.Net;
 using System.Reflection;
+using MaxMind.Db.Test;
```

Comment:
Fixed the namespace in `Program.cs` and removed this we have `Program.CurrentDirectory` defined there (abstracting out API differences).